package.json

```diff
@@ -0,0 +1,29 @@
+{
+  "name": "jp.wizcorp.phonegap.plugin.wizUtilsPlugin",
+  "version": "2.0.0",
+  "description": "This plugin allows you to access various app and device information, as well as copy/paste and restart app utilities.",
+  "cordova": {
+    "id": "jp.wizcorp.phonegap.plugin.wizUtilsPlugin",
+    "platforms": [
+      "ios",
+      "android"
+    ]
+  },
+  "keywords": [
+    "ecosystem:cordova",
+    "cordova-ios",
+    "cordova-android"
+  ],
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/thanhnhan2tn/phonegap-plugin-wizUtils"
```

Comment:
Would you mind updating the repository URL to this repo? :)

Reply:
That is using in my project, I cannot change it now.
You can merge and update that by your self.

Reply:
Updated